docs/core/testing/index.md

```diff
@@ -62,3 +62,5 @@ The XUnit team has written a tutorial that shows
 [how to use xUnit with .NET Core and Visual Studio](http://xunit.github.io/docs/getting-started-dotnet-core.html).
 
 If you prefer using MSTest, try the [walkthrough creating unit tests with MSTest and the .NET CLI](unit-testing-with-mstest.md).
```

Comment:
`walkthrough, [Unit testing with dotnet test and MSTest](...)`

Each of these items after the "Learn..." paragraph could be a bulleted list item; otherwise, they appear independent of one another.